### ISSUE TITLE
docs: cloud docs are hard to use

### ISSUE THREAD

**Issue by pnadolny13** [meltano/meltano#7677]
I find the cloud docs pretty hard to navigate right now, the top nav links me to the cloud page that only has the index https://docs.meltano.com/cloud/. I'm never quite sure where to go from there and how to find what I'm looking for.

Some ideas and suggestions:

- Add some kind of marketing blurb to the cloud docs page. What is cloud? Why use it over self hosted? Main features? How to get started today or get more info?
- Link to https://meltano.com/cloud/
- The cloud index page should probably have a "next" link to encourage users to go to the next appropriate page. I think that would be onboarding but not sure.
- Maybe add "references" as a subgroup that encapsulates stuff like: CLI, platform info, security
- Adding a new section related to Support or troubleshooting. Redirecting people to slack or where ever else.
- Remove https://docs.meltano.com/cloud/encrypting_secrets because I dont think its used anywhere
- "Meltano cloud" is an item in the index which links to the page youre already on

**Comment by tayloramurphy** [comment]
Agreed! PRs welcome 😄 We're also going to move to Docusaurus pre-GA which will help (but also make more evident any information architecture problems)

**Comment by pnadolny13** [comment]
Closed by https://github.com/meltano/meltano/pull/7758
